spiketools/utils/extract.py

```diff
@@ -0,0 +1,137 @@
+"""Utility functions for extracting data segments of interest."""
+
+import numpy as np
+
+###################################################################################################
+###################################################################################################
+
+def get_range(data, min_value=None, max_value=None, reset=None):
+    """Get a specified range from a vector of data.
```

Comment:
The function name `get_range` makes me think of the functionality of `compute_range`. 
What do you think about changing it to `get_value_in_range` or something like that? 
Also, the description could be more specific, maybe something like `"""Get values from a data array within a specified range.`?

Reply:
I'll keep this in mind and do a sweep for names before 0.1